libraries/botbuilder-ai/src/luisRecognizer.ts

```diff
@@ -205,6 +210,33 @@ export class LuisRecognizer {
         });
     }
 
+    private prepareErrorMessage(error: Error): Error {
+        switch ((error as any).response.statusCode) {
+            case 401:
```

Comment:
Missing 400. "request's parameters are incorrect meaning the required parameters are missing, malformed, or too large"

Reply:
Added this, good catch! #Resolved